lib/stdlib/doc/src/queue.xml

```diff
@@ -44,9 +44,18 @@
       not lists. Improper lists cause internal crashes.
       An index out of range for a queue also causes
       a failure with reason <c>badarg</c>.</p>
+      <p><em>Example:</em></p>
+      <pre>
+1> <input>queue:len([]).</input>
+** exception error: bad argument</pre>
 
     <p>Some functions, where noted, fail with reason <c>empty</c>
       for an empty queue.</p>
+    <p><em>Example:</em></p>
+      <pre>
+1> <input>queue:get(queue:new()).</input>
+** exception error: empty</pre>
+
```

Comment:
Are examples of invalid input really needed? :thinking:

Reply:
The sentence "fail with reason ..." is used in many places in the documentation with the meaning "throws an exception with reason ...", so I think it is generally understood. Even if it does need to be clarified, I think it would be better just to say "throws an exception" instead of giving an example.